### PR TITLE
Added unit testing for Android and updated syllabus

### PR DESCRIPTION
**Pull request by bentley-ritson-umuzi**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/projects/kotlin/project-10/unit-testing/_index.md
Comment (sheenarbw) [inline content/projects/kotlin/project-10/unit-testing/_index.md:7]: are we sure that there are no pre-requisites? Shouldn't they have done a bit of kotlin and android work?

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Looks good. The only thing is pre-requisites.

Also, the project is very hand-holdey/ spoon-feedy... Is there a plan for a more advanced project? 
Getting them to do the memory game or game of life could be cool in a seperate project becuase it has a visual element

File: content/projects/kotlin/project-10/unit-testing/_index.md
Comment (bentley-ritson-umuzi) [reply content/projects/kotlin/project-10/unit-testing/_index.md:7]: Yes. Unit testing stands on its own.

Comment (bentley-ritson-umuzi) [reply content/projects/kotlin/project-10/unit-testing/_index.md:7]: They basically do it when they are comfortable with the basics.

Comment (sheenarbw) [reply content/projects/kotlin/project-10/unit-testing/_index.md:7]: Hmm. If I go visit the tutorial though it assumes that I know a whole lot of stuff. Could someone do this project as their first card on their board? If that is the case then we probably have some missing cards - it looks like people would at least need to know a bit of kotlin syntax.

Students get to choose what they want to work on next. So long as a card is not blocked by an incomplete pre-requisite then they should be able to do the work. It really looks like some knowledge is needed here. 

If we were to ask someone like Sam to learn unit testing in kotlin, what cards would she NEED to complete before she would be ready to tackle this one?

Comment (bentley-ritson-umuzi) [reply content/projects/kotlin/project-10/unit-testing/_index.md:7]: I would probably leave it to the end. With all unit testing, the syntax is assumed knowledge. To go over exactly which topic is needed to be done is not the foundation for unit testing. I feel its better to have it on its own, then when the workshop is done, it covers all the concepts that are needed. If I was to force prerequisites, there will be assumed knowledge that will be left out.

Comment (bentley-ritson-umuzi) [reply content/projects/kotlin/project-10/unit-testing/_index.md:7]: If you think its a must then ill take time and go through the content in detail and add them. It's not a problem.

**Review by sheenarbw — APPROVED** [review]
Ok cool. I think pre-requisites are worth filling in, but it's not super urgent.

**Pull request merged**
